docs/storefront/graphql/examples/products.mdx

```diff
@@ -598,6 +598,159 @@ You can query product images at different resolutions. The following query retri
   </Tab>
 </Tabs>
 
+## Get product videos
+
+Query the product videos and all configurable media types for products. The API returns images uploaded in the control panel, image URLs added to a product using the control panel, and video URLs linked to a product. 
```

Comment:
Could this be simplified into one sentence? i.e. Fetch videos for a product, returning images and URLs added via the control panel or API.